docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+= Creating a group snapshot
+
+In {ProductName}, the integration service provides `group` snapshot to allow users to submit multiple related pull/merge requests under one application to their repos, have them tested together and have the CI results posted back to pull/merge requests.
+
+== Procedures
+To have `group` snapshot created for multiple pull/merge requests, complete the following steps:
+
+. Identify the application and components that are related and need to be updated.
+. Navigate to their git repos.
+. Create branches with the same branch name for all of these git repos.
+. Commit code changes to branches created above and create pull/merge requests.
+
+After every build pipelinerun succeeds and component snapshot is created, group snapshot will be created for the new built component container images and other components. And then integration test runs for component and group snapshot.
+
+NOTE: Group snapshot is created only when there are at least two components with open pull/merge request and the same source branch name. 
+
+The created group snapshot is like below:
+
+[source]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Snapshot
+metadata:
+  name: snapshot-sample
+  namespace: ws-sample-tenant
+  labels:
+    test.appstudio.openshift.io/type: group <1>
+    test.appstudio.openshift.io/pr-group-sha: bc3c81ad3d0140f3fe63c61a0b2389cb76663d226fbd28b00fb0d9f050fb21 <2>
+  annotations:
+    test.appstudio.openshift.io/pr-group: feature1 <3>
+    test.appstudio.openshift.io/group-test-info: [{"namespace":"ws-sample-tenant","component":"component-sample-1","buildPipelineRun":"component-sample-1-on-pull-request-****","snapshot":"application-sample-****","pullRuestNumber":"1","repoUrl":"https://github.com/sample-org/sample-repo1"},{"namespace":"ws-sample-tenant","component":"component-sample-2","buildPipelineRun":"component-sample-2-on-pull-request-****","snapshot":"application-sample-****","pullRuestNumber":"1","repoUrl":"https://github.com/sample-org/sample-repo2"}] <4>
```

Comment:
Are the astrixes just you excluding information from the sample CR?

Reply:
Yes, let me show real text. Thanks.